optaplanner-docs/src/main/asciidoc/QuickStart/QuickStart-chapter.adoc

```diff
@@ -14,3 +14,5 @@ include::Quarkus/QuarkusQuickStart-chapter.adoc[leveloffset=+1]
 include::SpringBoot/SpringBootQuickStart-chapter.adoc[leveloffset=+1]
 
 include::PlainJava/PlainJavaQuickStart-chapter.adoc[leveloffset=+1]
+
+include::VaccinationScheduler/VaccinationSchedulerQuickStart-chapter.adoc[leveloffset=+1]
```

Comment:
Let's move this under chapter 3, use cases and examples, such that first users don't face a paradox of choice.
The Cloud balancing tutorial is also being moved to chapter 3, for the same reason.